woocommerce/wc-admin
# use Jetpack Changelogger for changelog management

Fixes #6837 

This PR uses [Jetpack Changelogger ](https://github.com/Automattic/jetpack-changelogger)to improve merge conflict issues with changelogs.

### Question ###

@adrianduffell mentioned that it might be worth moving to a standard `CHANGELOG` file as our[ plugin page](https://wordpress.org/plugins/woocommerce-admin/) is getting longer and longer as we add more changelogs. I think that's really a good idea, but I'm not sure what the process would be. 

Should we delete the existing changelogs and make a link to the new `CHANGELOG` file in the `readme.txt`?

Do we want to use the [markdown format](https://keepachangelog.com/en/1.0.0/) or keep our plain text format?

### Detailed test instructions:

1. Clone this repository.
2. Create a few changelogs by running `npm run changelogger -- add`
3. Your changelog files should be created in the `./changelogs` directory.
4. Merge and write the changelog files into `changelog.txt` by running `npm run changelogger -- write
5. Check `changelog.txt` and confirm the changelogs.



no changelog need
